<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_memoryDeviceSetSubDialog</class>
 <widget class="QDialog" name="dialog_null_memoryDeviceSetSubDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_null_memoryDeviceSetSubDialog_device">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_null_memoryDeviceSetSubDialog_size">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_null_memoryDeviceSetSubDialog_used">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_null_memoryDeviceSetSubDialog_status">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
